src/reports/tables/ThreeOne.jsx

```diff
@@ -0,0 +1,231 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+
+const renderData = characteristics => {
+  return characteristics.map((characteristic, index) => {
+    return [
+      renderCharacteristicTitle(characteristic.characteristic, index),
+      renderCharacteristicDetails(characteristic)
+    ]
+  })
+}
+
+const renderCharacteristicTitle = (characteristic, key) => {
+  return (
+    <tr key={key}>
+      <th
+        colSpan={19}
+        style={{
+          borderTopWidth: '2px',
+          fontWeight: 'bold',
+          textTransform: 'uppercase',
+          backgroundColor: '#f1f1f1'
+        }}
+      >
+        {characteristic}
+      </th>
+    </tr>
+  )
+}
+
+const renderCharacteristicDetails = characteristic => {
+  if (characteristic.characteristic === 'Race')
```

Comment:
Rather than checking all the individual keys manually, you could do something [similar to this](https://github.com/cfpb/hmda-pub-ui/blob/a-and-d/src/reports/tables/TwelveOne.jsx#L40-L44). Would shorten the code up quite a bit. (you'll also need something [like this](https://github.com/cfpb/hmda-pub-ui/blob/a-and-d/src/reports/tables/TwelveOne.jsx#L40-L44) for the inner bit when looping through purchasers)